packages/components/checkbox/src/useCheckbox.ts

```diff
@@ -1,48 +1,18 @@
 import { ref, computed, inject, getCurrentInstance, watch } from 'vue'
```

Comment:
We should rename it to `kebab-case`.
Actually I think `hooks.ts` is better.

Reply:
I think hooks are divided according to their functions，so that may be `use-checkbox.hook.ts` will better?

Reply:
Or just rename it to `kebab-case`. How about `use-checkbox.ts`?

Reply:
i think it's a good idea, because the beginning of  `use` already means `hook`